Source/WebInspectorUI/UserInterface/Views/MediaDetailsSidebarPanel.js

```diff
@@ -317,14 +321,16 @@ WI.MediaDetailsSidebarPanel = class MediaDetailsSidebarPanel extends WI.DOMDetai
             && Object.shallowEqual(this.#values.video?.colorSpace, video?.colorSpace)
             && this.#values.video?.colorSpace?.primaries === video?.colorSpace?.primaries
             && this.#values.video?.colorSpace?.matrix === video?.colorSpace?.matrix
-            && this.#values.video?.fullRange === video?.fullRange)
+            && this.#values.video?.fullRange === video?.fullRange
+            && this.#values.video?.isProtected === video?.isProtected)
             return;
 
         this.#values.video = video;
         this.#ui.videoSection.element.classList.toggle("hidden", !video);
         this.#ui.resolutionRow.pendingValue = WI.UIString("%dx%d (%dfps)").format(video?.width ?? 0, video?.height ?? 0, video?.framerate ?? 0);
         this.#ui.resolutionRow.element.classList.toggle("hidden", !video);
         this.#ui.videoCodecRow.pendingValue = video?.codec;
+        this.#ui.videoProtectedRow.pendingValue = video?.isProtected ? WI.UIString("True") : null;
```

Comment:
the word "True" is a bit odd.  id suggest using ✔ (a.k.a. `"\u2713"`) like Web Inspector does in other places similar to this (e.g. cookie "Secure")